src/pybind/mgr/dashboard/frontend/src/app/shared/api/user.service.ts

```diff
@@ -40,6 +48,14 @@ export class UserService {
     });
   }
 
+  validateUserName(user_name: string): Observable<boolean> {
+    return this.get(user_name, true).pipe(mapTo(true), catchError(this.handleError));
+  }
+
+  handleError() {
+    return observableOf(false);
+  }
```

Comment:
```suggestion
  validateUserName(user_name: string): Observable<boolean> {
    return this.get(user_name, true).pipe(mapTo(true), catchError(_ => observableOf(false)));
  }
```

Reply:
Thank you. I knew there was a better way than what I did.